05/rodne_cislo_finish.py

```diff
@@ -0,0 +1,45 @@
+# Input ID number in Czech Republic
+# write ouf corect/incorect ID
+
+while True:
+    # MONTHS 30 DAYS
```

Comment:
V rámci while cyklu, ktorý beží kým užívateľ nezadá vstup odpovedajúci požiadavkom na formát by sa mali overovať len tieto požiadavky. Veškerá iná logika by mala byť vonku, nie je totiž nutné pri každom novom zadaní vstupu realizovať všetky kroky ktoré ti pripravujú data pre kontrolu (napr. riadok 5-19)